src/wp-includes/http.php

```diff
@@ -391,7 +389,7 @@ function wp_http_supports( $capabilities = array(), $url = null ) {
 		}
 	}
 
-	return (bool) $http->_get_first_available_transport( $capabilities );
+	return WpOrg\Requests\Requests::has_capabilities( $capabilities );
```

Comment:
Do we needs to mention these change through `@since` annotation?

Reply:
I did consider it, but it didn't feel like something that was worthwhile documenting.

From a developer perspective, nothing should change.

Reply:
Goatch.